src/hotspot/cpu/riscv/c2_MacroAssembler_riscv.cpp

```diff
@@ -1676,6 +1676,20 @@ void C2_MacroAssembler::signum_fp(FloatRegister dst, FloatRegister one, bool is_
   bind(done);
 }
 
+void C2_MacroAssembler::signum_fp_v(VectorRegister dst, BasicType bt, int vlen,
+                                    VectorRegister zero, VectorRegister one) {
+  vsetvli_helper(bt, vlen);
```

Comment:
Can we have a situation where vlen times sew(bt) won't fit into h/w register ?

Reply:
No, as UseRVV is only enable when vlenb >= 16, and `match_rule_supported_vector` return false if UseRVV == false.